standard/expressions.md

```diff
@@ -1025,6 +1070,19 @@ In case the parameter type sequences `{P₁, P₂, ..., Pᵥ}` and `{Q₁, Q₂
 - Otherwise if one member is a non-lifted operator and the other is a lifted operator, the non-lifted one is better.
 - If neither function member was found to be better, and all parameters of `Mᵥ` have a corresponding argument whereas default arguments need to be substituted for at least one optional parameter in `Mₓ`, then `Mᵥ` is better than `Mₓ`. Otherwise, no function member is better.
 
+#### §better-argument-passing-mode-new-clause Better argument-passing mode
+
+It is permitted to have corresponding parameters in two overloaded methods differ only by parameter-passing mode provided one and only one of the two parameters has value-passing mode, as follows:
```

Comment:
I don't think "only one" is required here, given that we're saying "they differ only in parameter-passing mode". They can't differ only in parameter passing mode and both have a value-passing mode :)